docs/dashboards/README.md

```diff
@@ -0,0 +1,237 @@
+    application -.-> sidecar-collector
+    end
+
+    gateway-collector:::collector-->selfmon-collector
+
+    sidecar-collector-.->Dynatrace
+    gateway-collector-.->Dynatrace
+
+    otherApps[Applications without sidecar collector]-.->gateway-collector
+    
+    style application fill:#adc9ff,stroke:#1966FF
+    classDef collector fill:#9afee0,stroke:#02D394
+    style sg-sidecar fill:#C2C2C2,stroke:#707070
+    style otherApps fill:#C2C2C2,stroke:#707070
+    style Dynatrace fill:#dcc2ff,stroke:#7F1AFF
+    style legend fill:#f2f2f2,stroke:#C2C2C2
+```
+
+Below, you can see a configuration example for a selfmonitoring collector.
+
+```yaml
+# receive selfmonitoring data via gRPC from OTel collector instances.
+receivers:
+  otlp/selfmon:
+    protocols:
+      grpc: 
+        endpoint: 0.0.0.0:4317
+
+processors:
+  # transform cumulative values to deltas. 
+  cumulativetodelta/selfmon: {}
+  # (kubernetes only) retrieves kubernetes attributes for other collectors sending to this collector. See Kubernetes prerequisites below.
+  k8sattributes/selfmon: {}
+
+  # prepend 'sfm.otelcol' to all selfmon metrics - the charts in the {dashboard_name}.json file expects this prefix.
+  transform/selfmon:
+    error_mode: ignore
+    metric_statements:
+      - context: metric
+        statements:
+          - set(name, Concat(["sfm.otelcol", name], "."))
+
+exporters:
+  # Inject DT_ENDPOINT and DT_API_TOKEN as environment variables. This should be the environment where the selfmonitoring data will go.
+  # See <https://docs.dynatrace.com/docs/shortlink/otel-getstarted-otlpexport> for instructions on which endpoint and token scope to use.
+  otlphttp/selfmon:
+    endpoint: "${DT_ENDPOINT}/api/v2/otlp"
+    headers:
+      Authorization: "Api-Token ${DT_API_TOKEN}"
+
+  # (optional) logs how many elements were exported to the Dynatrace backend.
+  debug:
+    verbosity: basic
+
+service:
+  # turn on the selfmonitoring for the selfmonitoring collector itself.
+  telemetry:
+    # (kubernetes only) the k8sattributesprocessor does not add attributes for the selfmonitoring collector itself. This is a known limitation of the processor.
+    # These environment variables need to be injected, see the Kubernetes prerequisite section below.
+    resource:
+      k8s.namespace.name: "${env:K8S_POD_NAMESPACE}"
+      k8s.pod.name: "${env:K8S_POD_NAME}"
+      k8s.node.name: "${env:K8S_NODE_NAME}"
+
+    metrics:
+      level: detailed
+      # export data via OTLP OTLP/grpc
+      readers:
+        - periodic:
+            interval: 20000
+            exporter:
+              otlp:
+                # the endpoint of the selfmonitoring collector. In this case, it is assumed that there is a service called `selfmon-collector` that exposes port 4317.
+                endpoint: selfmon-collector:4317
+                protocol: grpc/protobuf
+                temporality_preference: delta
+
+  extensions: []
```

Comment:
```suggestion
```

For me this line caused the following problem with helm:
`"Error: execution error at (opentelemetry-collector/templates/NOTES.txt:24:3): [ERROR] The opentelemetry-collector chart requires that the health_check extension to be included in the extension list."`